pkg/service/service.go

```diff
@@ -77,11 +78,12 @@ func (s simpleService) post(ctx context.Context, c card.Office365ConnectorCard,
 		return pr, err
 	}
 
-	req, err := http.NewRequestWithContext(ctx, "POST", s.webhookURL, bytes.NewBuffer(b))
+	req, err := retryablehttp.NewRequest("POST", s.webhookURL, bytes.NewBuffer(b))
 	if err != nil {
 		err = fmt.Errorf("failed to creating a request: %w", err)
 		return pr, err
 	}
+	req = req.WithContext(ctx)
```

Comment:
I'd prefer to keep it as it was by using the retryable http client like this in `main.go`:

```go
retryClient := retryablehttp.NewClient()
retryClient.RetryMax = 10

standardClient := retryClient.StandardClient() // *http.Client
```

Then you don't need to change anything else in the code. Apart from that, It's more natural to pass `ctx` as an input parameter. Fine for you?

Reply:
I was not sure if the standardClient would still consider the set retryMax. This is why I originally did it this way. However after reading the documentation thoroughly I came to the same conclusion as you and changed it accordingly.